portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -46,6 +51,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     double lat = Double.parseDouble(request.getParameter("lat"));
     double lng = Double.parseDouble(request.getParameter("lng"));
 
+    /** Checks for valid coordinates (limited area covered). */
+    if((lat < LAT_SOUTH|| lat > LAT_NORTH) || (lng < LNG_WEST|| lng > LNG_EAST)){
+        return;
```

Comment:
System.out.println('Outside of bounds') so you know what happened

Reply:
Yes, I had it but I removed it, I'll put it again, thanks!